explorePy/variable_summary.py

```diff
@@ -2,15 +2,57 @@
 import pandas as pd
 
 
-
-def variable_summary(x):
+def variable_summary(frames):
     """
     takes a data frame as input and provides the total quantity of each type of variable present in the data frame
-
     inputs:
-    x: data frame
-
+    frames: data frame
     returns:
     data frame - variable type and count of each type
     """
-    return x.variable_summary()
+    #https://stackoverflow.com/questions/19900202/how-to-determine-whether-a-column-variable-is-numeric-or-not-in-pandas-numpy
+    #pandas data types
+    #https://pbpython.com/pandas_dtypes.html
+
+    cols = list(frames.columns.values)
+    col_len = len(cols)
+
+
+    cols2 = frames.shape[1]
+
+    int_count = 0
+    char_count = 0
+    bool_count = 0
+    date_count = 0
+    other_count = 0
+
+    for i in range (0, col_len):
+        if frames[cols[i]].dtype == np.int64:
+        #if np.issubdtype(frames[cols[i]].dtype, np.number) == True:
```

Comment:
can we delete this line? (31)

Reply:
Whoopsie, 

you are right we do not need it.